src/main/java/com/jonathanbae/phototracker/PhotoRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.jonathanbae.triptracker.repository;
+
+import org.springframework.data.repository.CrudRepository;
+
+import com.jonathanbae.phototracker.Photo;
+
+public interface PhotoRepository extends CrudRepository<Photo, Integer> {
+    Photo findPhotoByName(String name);
```

Comment:
If you added this, either use it, or remove it if it's not used.

Reply:
this one i just copied the original triprepository because i wasnt sure if i needed it or not